controller/Dockerfile

```diff
@@ -3,20 +3,20 @@ ARG DEBIAN_VERSION="11.7"
 FROM --platform=$BUILDPLATFORM ubuntu:24.04 as builder
 
 RUN mkdir -p \
```

Comment:
Can use WORKDIR which creates and sets the dir? Otherwise lets just put it onto the same line.